### DISCUSSION TITLE
Jamulus headless server -> recordings

### DISCUSSION THREAD

**Discussion by Mickinator** [jamulussoftware/jamulus#1540] [Technical Support & Help]
Hi everyone,
this is my first posting on this site.
I´m running a private linux-server and it works well. Now I like to record our music, but I do not understand the explanations on the user jamulus page, because my english is too bad.

I understood, that I have to install a folder outside jamulus -> check (/recorder)
Now I have to enable recording in the startscript using -R "/recorder" -> check

But now I don´t know what´s to do. I do not understand exactly how to start and stop recordings and to store i#them in the directory /recording.

Can anyone help? I would be glad ;-)

**Comment by hoffie** [comment]
Hi!

> But now I don´t know what´s to do. I do not understand exactly how to start and stop recordings and to store i#them in the directory /recording.

Recordings should automatically start when the first user joins.

If you are using `--norecord` you will have to start recordings explicitly, either by using the GUI or by using signals on linux (USR2 to toggle recording).

If you've already tried that, it would make sense to look at the log output. Can you check and possibly share that?
